articles/100_ament.md

```diff
@@ -0,0 +1,168 @@
+But that is just a recommendation and not enforced by ament.
+
+Depending on where the packages are being installed it might be necessary to setup the environment in order to find all resources.
+E.g. the location of installed executables should be on the `PATH`, installed Python code should be on the `PYTHONPATH` etc.
+
+Therefore each package can provide a shell script to setup the environment to match its needs.
+The shells scripts in the install space, which are generated by the ament tools, will then only iterate over the scripts from each package to setup the environment.
+
+
+### Symlinked install space
+
+It is very important to make the development cycle of changing code, building and installing it and then run it to confirm the changes as efficient as possible.
+Commonly the installation steps involves copying some resources from the source space to their final destination in the install location.
+ament provides an option to create symbolic links instead (if the platform supports that).
+This enables the developer to change the resources in the source space and skipping the installation step in many situations.
+
+For CMake packages this is achieved by overriding the CMake `install()` function.
+For Python packages the [development mode[(http://pythonhosted.org//setuptools/setuptools.html#development-mode) is used to install the package.
+
+
+## How does *ament* relate to *catkin*?
+
+`catkin` has been used for ROS 1 since ROS Groovy.
+`catkin` is based around CMake and even packages only containing Python code are being processed via CMake.
+It was developed as a replacement of `rosbuild` which was used from the beginning of ROS.
+Therefore various features of catkin have been designed to be similar to rosbuild.
+
+
+### Why not continue to use *catkin*?
+
+catkin has many advantages over rosbuild.
+Those cover out-of-source builds, automatic CMake config file generation, an installation target and many more.
+But for some cases it is not flexible enough or not as easy to use as it should be.
+A few of these cases are mentioned below and all addressed by ament:
+
+
+#### *Devel space*
+
+catkin has the feature to provide a so called [devel space](http://wiki.ros.org/catkin/workspaces#Development_.28Devel.29_Space) after building a set of packages.
+That folder is providing a fully working ROS environment without the need to install the packages.
+Avoiding to copy any files allows to e.g. edit Python code and immediately try running the code.
+
+While this is a very convenient feature and speeds up the development process it comes at a cost.
+The necessary logic in catkin increases its complexity significantly.
+Additionally the CMake code in every ROS package has to make sure to handle the *devel space* correctly which puts an extra effort on every ROS developer.
+
+ament provides the same advantage using the option of *symlinked installs* without the extra complexity for each ROS package.
```

Comment:
I think if an outsider who has never used ament before reads this, they will be confused, then figure out what a symlinked install does from the name. I think also because "symlinked install" is boldfaced, I expected a definition of the term. It depends how much you want the readers to think for themselves. If this were a tutorial I would give an example file structure showing the install folder, etc. but for this document it's probably fine.